Cargo.toml

```diff
@@ -300,7 +300,7 @@ semver = "1.0.22"
 seqlock = "0.2.0"
 serde = "1.0.197"
 serde_bytes = "0.11.14"
-serde_derive = "1.0.103"
+serde_derive = "1.0.197"
```

Comment:
The linked issue mentions the need to keep them in sync during upgrades, which dependabot will likely miss, so can you add a comment saying that this needs to be kept in sync with the `serde` version?